src/scf-release/src/acceptance-tests-brain/test-scripts/minibroker_helper.rb

```diff
@@ -155,9 +157,13 @@ def run_test
                 elapsed = failed_service_creation - started_service_creation
                 raise "Failed to create service instance #{service_instance} after #{elapsed} seconds."
             end
+
+            STDERR.puts "#{c_bold}# Show instance...#{c_reset}"
```

Comment:
Not sure _this_ justifies extra bold output.  (I can see it for the "everything is ready" part though.)

Reply:
I wanted markers which are easily visible in the log when read by a user.
That meant to me to put them on the same level as the output from the `run` commands.
I guess I could go colored instead ... blue ?

Reply:
Done with commit e8a0eca19f6e6b69f9f685bda659c9a87380d02c.